src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -170,6 +175,17 @@ export class MdAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
     }
   }
 
+  get outsideClickStream(): Observable<any> {
```

Comment:
Hmm let's keep this private?  I think I may have omitted the _ in my comment above by mistake, but I liked that you originally had it private.